persistence/nhibernate/outbox_transactionisolation_nhibernate_[8.5,).partial.md

```diff
@@ -4,4 +4,4 @@ The transaction isolation level for the outbox operation can be specified:
 
 snippet: OutboxTransactionIsolation
 
-Note: The default isolation level is `Serializable`. The isolation level values of `Chaos`, `ReadUncommitted`, `Snapshot` and `Unspecified` are not allowed.
\ No newline at end of file
+Note: The default isolation level is `Serializable`. The isolation level values of `Chaos`, `ReadUncommitted`, `Snapshot` (outbox relies on pessimistic locking to prevent more-than-once invocation) and `Unspecified` are not allowed.
```

Comment:
@ramonsmits does it make sense to state it like so? 

```suggestion
Note: The default isolation level is `Serializable`. The isolation level values of `Chaos`, `ReadUncommitted`, `Snapshot` and `Unspecified` are not allowed. Outbox relies on pessimistic locking to prevent more-than-once invocation.
```

Reply:
How about adding the word concurrent too:
```suggestion
Note: The default isolation level is `Serializable`. The isolation level values of `Chaos`, `ReadUncommitted`, `Snapshot` and `Unspecified` are not allowed. Outbox relies on pessimistic locking to prevent concurrent more-than-once invocation.
```